### ISSUE TITLE
Allow ignoring Pre-Releases

### ISSUE THREAD

**Issue by amogus07** [marwanhawari/stew#44]
Right now, I use stew to keep the syncthing binary up-to-date. Today, `stew upgrade --all ` updated it to v1.27.10-rc.1, which is a labeled as "Pre-Release". Is there a way to have stew only pull updates labeled as "Latest" and ignore Pre-Releases?

**Comment by marwanhawari** [comment]
I agree, the only time I would want to see prereleases are for the `stew browse` command. Everything else should just pull the latest non-prerelease release.

Looks like there is a boolean `prerelease` property [returned in the HTTP response](https://docs.github.com/en/rest/releases/releases?apiVersion=2022-11-28#list-releases) for the GitHub API `GET /repos/{owner}/{repo}/releases` endpoint. Would need to update [this type](https://github.com/marwanhawari/stew/blob/79280da92eea04ee41264eb8d265e07230955394/lib/github.go#L24) and then update the application code to filter out prerelease releases (except for `stew browse`).

Do you want to try and take this on?

**Comment by amogus07** [comment]
I'm sorry, I would love to help out, but I don't know Go, neither am I familiar with this project's source code. As of now, someone else will have to do it.

**Comment by HugoDF** [comment]
@marwanhawari if this is still of interest I can work on a Pull Request

**Comment by marwanhawari** [comment]
Yeah I'd be happy to review a PR!

**Comment by HugoDF** [comment]
@marwanhawari PR is ready for review https://github.com/marwanhawari/stew/pull/49
